app/models/price-profile.js

```diff
@@ -19,6 +20,11 @@ export default AbstractModel.extend({
       numericality: {
         allowBlank: true
       }
-    }
+    },
```

Comment:
Travis is failing tests due to formatting issues.  Please run ember test to check the formatting issues locally.
```
not ok 736 Chrome 53.0 - JSCS - models/price-profile.js: should pass jscs
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:25353:12)
                at runTest (http://localhost:7357/assets/test-support.js:2995:28)
                at Object.run (http://localhost:7357/assets/test-support.js:2980:4)
                at http://localhost:7357/assets/test-support.js:3143:11
                at process (http://localhost:7357/assets/test-support.js:2819:24)
                at begin (http://localhost:7357/assets/test-support.js:2801:2)
        message: >
            models/price-profile.js should pass jscs.
            disallowTrailingWhitespace: Illegal trailing whitespace at models/price-profile.js :
                26 |        allowBlank: true
                27 |      }
                28 |    }    
            -------------^
                29 |  }
                30 |});
        Log: |
```